frontend/model/contracts/group.js

```diff
@@ -229,14 +230,16 @@ DefineContract({
     'gi.contracts/group/invite': {
       validate: objectOf({
         inviteSecret: string, // NOTE: simulate the OP_KEY_* stuff for now
-        numInvites: number
+        numInvites: number,
+        invitee: string
       }),
       process (state, { data, meta }) {
         Vue.set(state.invites, data.inviteSecret, {
           generated: data.numInvites,
           creator: meta.username,
           responses: {},
-          status: 'valid'
+          status: 'valid',
```

Comment:
These values should come from constants defined somewhere. So add a TODO here, like `// TODO: define all possible values as constants`

Reply:
Fixed!